core/CMakeLists.txt

```diff
@@ -158,7 +158,9 @@ if(XCODE OR VS)
 endif()
 
 message("CMake ${_AX_CORE_LIB} target_precompile_headers")
-target_precompile_headers(${_AX_CORE_LIB} PRIVATE "$<$<COMPILE_LANGUAGE:CXX>:cocos2d.h>")
+target_precompile_headers(${_AX_CORE_LIB} PRIVATE 
+  "$<$<COMPILE_LANGUAGE:CXX>:cocos2d.h>"
```

Comment:
@crazyhappygame Should `axmol.h` be added to this list given that, moving forward, it's the one we should be using?